keymaker/__init__.py

```diff
@@ -37,7 +37,32 @@ def configure(args):
     print("Will configure", args)
 
 def get_authorized_keys(args):
-    iam = boto3.client("iam")
+    session = boto3.Session()
+    iam = session.client("iam")
+    sts = session.client("sts")
+    config = {}
+    try:
+        role_arn = parse_arn(sts.get_caller_identity()["Arn"])
+        _, role_name, instance_id = role_arn.resource.split("/", 2)
+        for role_desc_word in iam.get_role(RoleName=role_name)["Role"]["Description"].split():
+            if role_desc_word.startswith("keymaker_") and role_desc_word.count("=") == 1:
```

Comment:
This probably works fine for now, but in order to make evolving and using this a bit easier, should we consider json or yaml?

Reply:
specifically, encoding the data in the description as json

Reply:
Actually, even yaml seems impossible since `:` is not allowed either ☹️

Reply:
I could make this more "natural" by parsing syntax like `key1 = value1, key2 = value2`. i.e. split on comma as well as space, then split on space as well as `=`

Reply:
that that's probably a fine a compromise. :(